### PR TITLE
Add procurement principles

### PR DESCRIPTION
**Pull request by bvhme**
Input more than welcome

### CODE REVIEW

**Comment by bvhme** [comment]
Mirjam has suggested adding:

- [ ] Examples
- [ ] Criteria to test our purchases

And perhaps 'buying terms' in the future, however I feel we probably don't want that kind of overhead at the moment.

File: activities/procurement/principles.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+type: Resource
+---
+
+# Procurement principles
+
+To fulfill our mission, we need to be able to be open, reliable, trustworthy, helpful and sustainable. Meaning the services and products we use need to be:
```
Comment (ElenaFdR) [inline activities/procurement/principles.md]:
```suggestion
To fulfill our mission, we need to be able to be open, reliable, trustworthy, helpful and sustainable. This means the services and products we use need to be:
```

**Comment by ElenaFdR** [comment]
@bvhme @MirjamvT If we add more detail (in the future) it would make sense to;
- align our procurement rules to those of our members, since we'll effectively be spending taxpayer money
- make this very explicit so that it's clear we're not engaging in cronyism or other form of public corruption

**Comment by ElenaFdR** [comment]
Re more future detail, see #372.

**Pull request merged**